frontend/public/style/_common.scss

```diff
@@ -238,6 +238,18 @@
   @media (min-width: $screen-sm-min) {
     margin: $grid-gutter-width;
   }
+
+  .co-well-header button {
```

Comment:
We should have specific styles for a start guide close button, not change the position of buttons in all start guides.